### DISCUSSION TITLE
A Guide to Getting the Most out of VC

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#181] [Ideas]
When @mrsantons and I met this weekend, we talked about creating a Guide to Getting the Most out of VC.

I think this could be a great way to provide some central info, but also to help establish what type of community we are. There are many slack communities where people drop in, ask for help, and leave. I think we do a pretty good job of showing what type of community we are, but I'd like to make this as clear as possible. Even for non-members. This might be a place we can direct people to, to find out more about VC.

I don't know if this should be tackled on the site in our about page, or in some other way. But I'd love to have thoughts on how and/or what we should include.

**Comment by ClJarvis** [comment]
I think that could be beneficial for new members or recruiting new members. The About page would be a good place for it. Create a sub heading on that page. I'd be afraid to add it as a separate page. if there's too much on the Navbar it makes a site clunky. 

It could also be added to the newsletter. Either as a snippet of the main guide with a link back or just a short version. Or a rotating clip if it somehow matched the monthly challenge.  With a sample of how members helped each other during the challenge.

**Comment by tkshill** [comment]
More and better onboarding sounds awesome. I immediately thought something available for slack members but since people meet us in zoom before slack, and the zooms are a core part of VC, it can see the use case for having it on somewhere online. 

I think a starter template can be

What to expect
- In the zoom
- In the slack
    - How to participate in channel discussions
    - How to find relevant channels
    - Channel curation
- How to do events
    - What are the monthly challenges
    - What are Lunch n' Learns
    - What else is going on
- How to volunteer
- How to contribute
- How do I keep up
   - The podcast
   - The newsletter
- Where do I go for help
    - The forms
    - The maintainers
- F.A.Q.

**Reply by BekahHW** [reply]
This is great!

**Comment by BekahHW** [comment]
Ok, where do we go from here? 

I'm almost inclined to make this a google doc to make it easier for everyone to contribute to, and then move it to an official place (like the about page or something) when we feel it's in a solid place

**Comment by BekahHW** [comment]
@ClJarvis are you interested in taking the lead on this or on any of the discussion issues you've commented on?

**Reply by ClJarvis** [reply]
Like building the guide as info is collected? Sure, I'd like to do that.

**Reply by BekahHW** [reply]
Awesome. Let me know how you'd like to do that. We could have a form that we put out in slack, or we can have a google doc, or an open issue that ppl comment on. I'd love to have your input on this.

**Comment by ClJarvis** [comment]
I'm trying to think what would be better. A form would have everything in a single document. But members would need to see other members replies to have more of a conversation.

Can anyone comment on an open issue?

**Reply by BekahHW** [reply]
No in this repo, so we'd have to move it to VC.io repo, which I'm not opposed to. Updating the content on the site has been in my todo list for a while.

**Reply by ClJarvis** [reply]
If we use github issues that could let newer devs have more exposure to this function of github especially before we begin Hacktoberfest. 
I found last year Hacktoberfest instructions clear, but would this help the very new devs members did they have trouble with gitHub last year.
Otherwise a form shared in slack seems like the best option.

**Reply by BekahHW** [reply]
@ClJarvis Ok, let's go for the VCIO repo for the issue. Do you want to write the issue?

We want:
- An explanation of the purpose of the issue -- to create a guide that can be used as a member resource;
- An invitation for people to contribute by adding things in the comments;
- Maybe some Category Type headings to start off with.

**Reply by ClJarvis** [reply]
@BekahHW Is that the pinned repo? 

I can start on it soon. Kirk has a great outline to build on.

**Reply by BekahHW** [reply]
@ClJarvis yep. The pinned one. Here's the link as well: [https://github.com/Virtual-Coffee/virtualcoffee.io](https://github.com/Virtual-Coffee/virtualcoffee.io)

**Reply by ClJarvis** [reply]
I opened an Issue. https://github.com/Virtual-Coffee/virtualcoffee.io/issues/293#issue-970650949

**Reply by danieltott** [reply]
This is great @ClJarvis ! 

If I could add one suggestion - add some direct actions people can take. What do you want people to do when they see this issue? I assume it's "add comments" - if so I'd add some language about that. I've learned it's important to be 1000% more specific than you think you need to be 😄

**Reply by ClJarvis** [reply]
I edited it. Will probably have to edit it more but that's part of documentation process.

**Comment by ClJarvis** [comment]
After looking at the amount of information the guide will cover, I think it does need its own page as opposed to being tacked on to another page. I'm currently looking through the repo to understand the file structure and 11ty.

**Reply by danieltott** [reply]
I agree 100%. In fact I was thinking about this, and I think it'd be great to have a whole new section called something like "Member Resources" with this as the first page - we could also add a channel guide and some other things down the line.

**Reply by danieltott** [reply]
So to be specific - let's add `src/member-resources/index.njk` (or `.md` if you want to roll with markdown) which will be the Member Resources home page, and `src/member-resources/guide-to-vc.njk` with the new content. I can help with this if you want to pair up tomorrow during the day!

**Reply by ClJarvis** [reply]
I like adding the member resources directory. That will give us room to build in the future.

What is the design reason to use MD verse .njk? is cause njk is filled by javaScript?

**Comment by ClJarvis** [comment]
I like adding the member resources directory.  That will give us room to build in the future. 

What is the design reason to use MD verse .njk? is cause njk is filled by javaScript?

**Reply by danieltott** [reply]
If you use markdown you can just write text in markdown and don't have to worry about tags and everything, but you don't have a lot of options. So - markdown is more simple but less options, nunjucks is more complicated but more flexible. The index should probably for sure be in njk now that I'm thinking about it - the guide can probably be either? I'd say go with whatever you're comfortable with to start. Can always change later.

**Reply by ClJarvis** [reply]
Ok more flexible is usually better.

**Comment by ClJarvis** [comment]
Is there a style guide or set process for naming files?  I'm taking guidance from Dan's instructions on how to add a new member issue and in this issue.  I've made basic pages for src/member-resources/index.njk and src/member-resources/guide-to-vc.njk. 

What hex value is used for the undraw illustrations?

**Reply by BekahHW** [reply]
This should help find the hex values: [https://github.com/Virtual-Coffee/virtualcoffee.io/blob/main/src/assets/scss/_variables.scss](https://github.com/Virtual-Coffee/virtualcoffee.io/blob/main/src/assets/scss/_variables.scss)

And all of the svgs are [here](https://github.com/Virtual-Coffee/virtualcoffee.io/tree/main/src/_includes/svg)

**Comment by BekahHW** [comment]
@adiati98 has said she could probably offer some support. She's on vacation now, but I wanted to make sure she's looped into this conversation.

**Reply by adiati98** [reply]
Thanks, @BekahHW!
I just started to catch up 😄

**Comment by danieltott** [comment]
I've generally used the value for `$pink`: `#d9376e`

I'm not sure what you're looking for as far as a process for naming files...

**Reply by ClJarvis** [reply]
For naming it looks like you used hyphens like "guide-to-vc." as opposed to under scores or camel case.

**Reply by danieltott** [reply]
Ah I see. Yeah - in 11ty, file names become the URIs by default. This can be overwritten in frontmatter, but it's easiest to go with the default behavior. Dashes and lower case are in general good defaults for web address URIs 👍

**Comment by ClJarvis** [comment]
While brainstorming the VC guide an FAQs was mentioned. If that was done now would it help the maintainers come October? 

- Should FAQ be one the VC guide or its own page under member resources?
- What were the FAQs from last year's HacktoberFest?
- What were the community maintainer asked the most last year? If we put that list up it could save time later.

**Comment by adiati98** [comment]
Hey @ClJarvis,
I'm here to support :)

I don't quite understand what to add for the HacktoberFest yet, but in general, would we make a page of documentation/guide about how to make the most out of Virtual Coffee through channels in Slack, Lunch 'n Learn, etc.? 
Should I compile things in, let's say, Google Docs following the template from @tkshill for now?

@BekahHW what contributions do we expect from members as you mentioned [here](https://github.com/Virtual-Coffee/VC-Contributors/discussions/181#discussioncomment-1160267)?

I'm trying to understand and catch up and be on the same page 😄
Thank you!

**Reply by ClJarvis** [reply]
@adiati98 Current overview, I have made a memebers-resources folder witha page for the VC Guide. 
It is on a pull request here. https://github.com/Virtual-Coffee/virtualcoffee.io/pull/298 I've used @tkshill 's outline and have content for much of it. We can add more detail to it. 
Need more detail on slack channels. 

- How does CoWorking room work
-  How does Book Club work. 
- How does the interview study group work.

For Hacktoberfest FAQs I thought that the maintainers may have had a bunch frequent questions asked last year as people joined VC. If we added them to memeber-resources that would help.

For How to stay update section: right now it is links to Newsletter, podcasts, twitter, and youtube. Do we need more? Is there any newsletter  or podcast we want to point out in particular?

**Reply by adiati98** [reply]
Great!
Let me take a look at the draft, and I will ask around about the Book Club because I'm not in there 😄
I know how Coworking room and interview study group work. But I will ask for more detail about the interview study group from Marie & Julia.

As for HacktoberFest, maybe @BekahHW & @tkshill could give some insight?

**Reply by BekahHW** [reply]
@adiati98 if we have something up on the website, we can potentially break the issue down into smaller issues. 

If that happens, then we can promote this as part of our Hacktoberfest initiative. 

These would be similar in difficulty to what you've done with the newsletter--at least I think. And we have a number of members who could provide guidance if needed.

**Comment by adiati98** [comment]
@ClJarvis, 
I cloned your VC repo, but the branch can't be cloned.
So I can't see your PR in my local repo.
That's why I write down the guides on Google Docs.
This is not yet finished and probably needs much editing.
But it would be great if I could have some feedback on this [Guide To Virtual Coffee](https://docs.google.com/document/d/18M_67XZeCSnb343KLjbkJbPuCe_GqN_6ZOyaBm0p60c/edit) 😊

**Comment by ClJarvis** [comment]
@adiati98 I like how you listed the ways to find and join slack channels.  I like the descriptions of the channels.
I think the branch will be added to the main repo soon. That will make it easier to add sections at a time. Dan dose want too big of a Pull Request at once.

> ----- I’m not sure whether we will include this or not. Would this be for members only, or also for people who will join VC?
 
> ----- Note: If you’re new, you need to attend one of the Coffees for you to be a member of Virtual Coffee. We want to know you and we would like to give you the experience of Virtual Coffee in person before we move to Slack Group

Yes we need something like this. The guide will be on the public facing site so people that want to join can learn more about VC.

**Comment by adiati98** [comment]
@ClJarvis I've updated the [Google Docs](https://docs.google.com/document/d/18M_67XZeCSnb343KLjbkJbPuCe_GqN_6ZOyaBm0p60c/edit) and I wrote some comments or questions in red 😊

@BekahHW @tkshill @danieltott would you mind checking the docs and give some inputs, especially the events, volunteering, contributing sections, and the comments?
Also, what else needs to be added to FAQ?

Thanks!

**Comment by danieltott** [comment]
@adiati98 @ClJarvis this is really great!!

I think I'm going to pull in some of Ayu's changes into the main PR and get it live - once that's done, we can do individual Issues and PRs to add/improve on it. Thank you both so much for this so far - I'm really excited about having this section up.
